syntax = "proto3";

package smaug;

import "node.proto";

message GraphProto {
  string name = 1;
  repeated NodeProto nodes = 2;
  // The backend that this graph should run on
  string backend = 3;
}
